resource/ckb.toml

```diff
@@ -88,16 +88,37 @@ cell_output_cache_size  = 128
 
 # Set the lock script to protect mined CKB.
 #
+# CKB uses CS architecture for miner. Miner process (ckb miner) gets block
+# template from the Node process (ckb run) via RPC. Thus the lock script is
+# configured in ckb.toml instead of ckb-miner.toml, and the config takes effect
+# after restarting Node process.
+#
 # The `code_hash` identifies different cryptography algorithm. Read the manual
 # of the lock script provider about how to generate this config.
 #
 # CKB provides an secp256k1 implementation, it requires a public key hash in
-# the args.
+# the args. The hash algorithm is blake2b, with personal "ckb-default-hash".
+# The first 160 bits (20 bytes) are used as the only arg.
+#
+# The public key is compatible with Bitcoin, use any tool you trust to generate
```

Comment:
"The public key is compatible with Bitcoin" is quite confusing to me.

my advise:

You can use any tool you trust to generate a Bitcoin private key and public key pairs, which can be used in CKB as well. CKB CLI provides the function for you to convert your public key into block assembler configuration parameters.

Here is an example using OpenSSL to generate the key pair:
```
....
```

You can then generate the block assembler lock with:
```
.....
```

Here is an example output:
```
.....
```